src/utility.ts

```diff
@@ -4,10 +4,12 @@ export const clean = (input: string): string => {
   return truncate(input.replace(/[^a-z0-9+]+/gi, ''))
 }
 
+const md5 = (input: string): string => {
+  return createHash('md5').update(input).digest('hex')
+}
+
 export const truncate = (input: string): string => {
-  return input.length <= 64
-    ? input
-    : input.substr(0, 32) + createHash('md5').update(input).digest('hex')
+  return input.length <= 64 ? input : input.substr(0, 32) + md5(input)
```

Comment:
Lets remove the md5 function, and leave this utility as is.

Reply:
https://github.com/neiman-marcus/serverless-provisioned-concurrency-autoscaling/issues/47